test/dynamo/test_misc.py

```diff
@@ -187,6 +187,24 @@ def fn(x):
             self, fn, 1, expected_ops=1, expected_ops_dynamic=10
         )
 
+    def test_int_shape_inplace_binops(self):
+        def fn(x):
+            p = x.shape[0]
+            # Test reversal by putting int first
+            y = 2
+            y += p
+            y -= p
```

Comment:
It's not clear to me that the latter tests necessarily do anything, because at that point y is no longer a constant and so it won't get printed inline.

Reply:
You're right - changed this to ensure we're working with a constant each time.